.changesets/breaking_lrlna_remove_deprecated_interfaces.md

```diff
@@ -26,6 +26,8 @@ succeeded.
 - `apollo_router_deduplicated_subscriptions_total`. Use the
 `apollo.router.operations.subscriptions` metric's `subscriptions.deduplicated`
 attribute
+- `apollo_router_span` has been removed. Use `apollo.router.processing.time`
```

Comment:
We should not recommend this either. We should instead remove processing time also. It's a flawed metric.

Reply:
```
Calculating the overhead of injecting the router into your service stack when making multiple downstream calls is a complex task. We are removing these misleading metrics until we can figure out something better.

It is recommended that you test your workloads with and the router and see if the latency meets your requirements.
```